x/btcstaking/keeper/power_dist_change.go

```diff
@@ -263,11 +263,11 @@ func (k Keeper) ProcessAllPowerDistUpdateEvents(
 		if _, ok := slashedFPs[fpBTCPKHex]; ok {
 			fp.IsSlashed = true
 
-			statusChangeEvent := types.NewFinalityProviderStatusChangeEvent(fp.BtcPk, types.FinalityProviderStatus_STATUS_SLASHED)
+			statusChangeEvent := types.NewFinalityProviderStatusChangeEvent(fp.BtcPk, types.FinalityProviderStatus_FINALITY_PROVIDER_STATUS_SLASHED)
 			if err := sdkCtx.EventManager().EmitTypedEvent(statusChangeEvent); err != nil {
```

Comment:
Not sure this is good to use `EmitEvent` because cosmos has listed `EmitEvent` as deprecated https://github.com/cosmos/cosmos-sdk/blob/v0.50.6/types/events.go#L45-L49